lib/experimental/script-modules.php

```diff
@@ -201,6 +201,43 @@ function gutenberg_dequeue_module( $module_identifier ) {
 	wp_script_modules()->dequeue( $module_identifier );
 }
 
+/**
+ * Prints HTML for the a11y Script Module.
+ *
+ * a11y relies on some DOM elements to use as ARIA live regions.
+ * Ideally, these elements are part of the initial HTML of the page
+ * so that accessibility tools can find them and observe updates.
+ */
+function gutenberg_a11y_script_module_html() {
+	$a11y_module_available = false;
+
+	$get_marked_for_enqueue = new ReflectionMethod( 'WP_Script_Modules', 'get_marked_for_enqueue' );
+	$get_marked_for_enqueue->setAccessible( true );
+	$get_import_map = new ReflectionMethod( 'WP_Script_Modules', 'get_import_map' );
+	$get_import_map->setAccessible( true );
+
+	foreach ( array_keys( $get_marked_for_enqueue->invoke( wp_script_modules() ) ) as $id ) {
+		if ( '@wordpress/a11y' === $id ) {
+			$a11y_module_available = true;
+			break;
+		}
+	}
+	if ( ! $a11y_module_available ) {
+		foreach ( array_keys( $get_import_map->invoke( wp_script_modules() )['imports'] ) as $id ) {
+			if ( '@wordpress/a11y' === $id ) {
+				$a11y_module_available = true;
+				break;
+			}
+		}
+	}
+	if ( ! $a11y_module_available ) {
+		return;
+	}
+	echo '<p id="a11y-speak-intro-text" class="a11y-speak-intro-text" style="position: absolute;margin: -1px;padding: 0;height: 1px;width: 1px;overflow: hidden;clip: rect(1px, 1px, 1px, 1px);clip-path: inset(50%);border: 0;word-wrap: normal !important;">' . esc_html__( 'Notifications', 'default' ) . '</p>';
```

Comment:
@afercia Do you know if there's a reason to use `p` element for the "Notifications" intro text instead of `div` for everything?

It doesn't seem like these elements have semantic value and generic `div` containers could be used for everything.

Reply:
The "Notifications" intro text was added because screen reader users can navigate any content of the page (not only focusable elements). As such, when a speak message 'stays' in the aria live region there is a change screen reader users may navigate to it and that wouldn't make sense because the message could be out of date or no more relevant. We considered to 'clear out' the live regions after some interval but that woul dbe completely arbitrary and unreliable. At the end we opted to leave the messages in place even though they may be no more relevant and add some text to explain what these messages are. That's why we added the `a11y-speak-intro-text` 'Notifications'. As per the element to use, a paragraph is always more appropriate for text as `div` element should only be used for layout (the `div` tag name actually means 'division').